circle.yml

```diff
@@ -163,8 +163,9 @@ defaults:
   evmc-test: &evmc-test
     run:
       name: "Run evmc tests"
+      working_directory: ~/build
       command: |
-        ~/build/evmc/test/evmc-vmtester ~/build/src/libhera.so
+        ctest
```

Comment:
Where is this name coming from?

Reply:
https://cmake.org/cmake/help/latest/manual/ctest.1.html